__tests__/endpoint.test.js

```diff
@@ -171,4 +171,48 @@ describe('Endpoint Tests', () => {
               });
           });
         });
+        describe('PATCH /api/articles/:article_id', () => {
+            test('200: updates article.votes by reciveing object and article id, then responds to client with updated article', () => {
+                const update = {inc_votes: 50}
+                return request(app)
+                .patch('/api/articles/1')
+                .send(update)
+                .expect(200)
+                .then(({body}) => {
+                    expect(body.updatedArticle).toBeInstanceOf(Object)
```

Comment:
I think this expect statement is slightly overkill. The expect below is implicitly showing that an object is coming back 👍